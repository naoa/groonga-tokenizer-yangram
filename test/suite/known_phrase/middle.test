register tokenizers/yangram

table_create known_phrases TABLE_PAT_KEY ShortText
load --table known_phrases
[
{"_key": "【請求項2】"}
]
tokenize TokenYaBigram "検索装置【請求項２】検索システム" NormalizerAuto --mode ADD
tokenize TokenYaBigram "検索装置【請求項２】検索システム" NormalizerAuto --mode GET
