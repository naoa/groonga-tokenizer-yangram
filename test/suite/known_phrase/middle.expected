register tokenizers/yangram
[[0,0.0,0.0],true]
table_create known_phrases TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
load --table known_phrases
[
{"_key": "【請求項2】"}
]
[[0,0.0,0.0],1]
tokenize TokenYaBigram "検索装置【請求項２】検索システム" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "検索",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "索装",
      "position": 1,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "装置",
      "position": 2,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "置",
      "position": 3,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "【請求項2】",
      "position": 4,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "検索",
      "position": 5,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "索シ",
      "position": 6,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "シス",
      "position": 7,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "ステ",
      "position": 8,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "テム",
      "position": 9,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "ム",
      "position": 10,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
tokenize TokenYaBigram "検索装置【請求項２】検索システム" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "検索",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "装置",
      "position": 2,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "【請求項2】",
      "position": 4,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "検索",
      "position": 5,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "シス",
      "position": 7,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "テム",
      "position": 9,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
